azure-kusto-data/azure/kusto/data/client.py

```diff
@@ -191,15 +200,25 @@ def execute_mgmt(self, database: Optional[str], query: str, properties: Optional
         """
         database = self._get_database_or_default(database)
         Span.set_query_attributes(self._kusto_cluster, database, properties)
-
-        return self._execute(self._mgmt_endpoint, database, query, None, self._mgmt_default_timeout, properties)
+        request = ExecuteRequestParams.from_query(
+            query,
+            database,
+            properties,
+            self._request_headers,
+            self._mgmt_default_timeout,
+            self._mgmt_default_timeout,
+            self._client_server_delta,
+            self.client_details,
+        )
+        return self._execute(self._mgmt_endpoint, request, properties)
 
     @distributed_trace(name_of_span="KustoClient.streaming_ingest", kind=SpanKind.CLIENT)
     def execute_streaming_ingest(
         self,
         database: Optional[str],
         table: str,
-        stream: IO[AnyStr],
+        stream: Optional[IO[AnyStr]],
+        blob_url: Optional[str],
```

Comment:
Technically a breaking change - since this function is "public" +
We probably need to reflect the change in the async version, even if we don't use it in ingest (since they need to have api compatibility )